ekingan/day2.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+
+  <!-- set viewport to device width to allow responsiveness -->
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <!-- Bootstrap CDN -->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+
+  <!-- custom styles (always require after Bootstrap) -->
+  <link rel="stylesheet" href="style.css">
+
+  <title>Homework day 2</title>
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="col-sm-6">
+        <h1>Questions I have about the homework</h1>
+        <ul>
+          <li><p>
+          I do not understand what a ternary operator is or what it is used for.
+          </p>
+          </li>
+          <li><p>
+          I am not always clear when to use a "while" loop.
```

Comment:
sometimes there's more than one way to do something; but this will also become a bit clearer soon I think